meetings/roles.md

```diff
@@ -0,0 +1,138 @@
+This document describes the various roles involved with running the meetings involved in the production of the learning path.
+All of these roles should be filled and completed for each meeting to be as successful as possible.
+The roles are:
+
+* [Scheduler]
+* [Crier]
+* [Facilitator]
+* [Scribe]
+
+<a name="scheduler"></a>
+## Scheduler
+
+The _Scheduler_ sends the calendar invite for the meeting.
+
+**Within 24 Hours of Previous Meeting**
+* [ ] Create an agenda document for the next meeting by copying this [template][template] into the [learning path folder].
+* [ ] Open the document and follow the steps in the **Do these things to set up this document** section.
+* [ ] Send the calendar invite for the next meeting.
+This invite must include:
+
+  * Conference call info.
+  * Comment-enabled link to the agenda document.
+
+### Tips
+
+* The _Scribe_ should have included the time of the meeting in the notes from the previous meeting. 
+* Include everyone from the previous meeting on the invite for the next meeting.
+An easy way to do this using _Outlook_ is to reply by meeting to the notes email from the previous meeting.
```

Comment:
There are other PIM programs, and likely most of them contain the _"Reply with meeting"_ button and can display the initial email (At least Evolution and Outlook do) - thus I would phrase this more generally 
```suggestion
An easy way to do this is to use the _Reply with meeting_ function to the notes email from the previous meeting.
```